test/cache-driver.js

```diff
@@ -231,7 +232,7 @@ suite('CacheDriver', function() {
 
         var otherAuthority = 'someOtherAuthority';
         var responseOptions = { authority : otherAuthority, mrrt : true, resource : responses[0].resource };
-        var differentAuthorityResponse = util.createResponse(responseOptions);
+        var differentAuthorityResponse = util.createResponse(responseOptions, 21);
```

Comment:
:) it's 21 because of the specific test case, we add 20 entries into cache, and then add another 2 entries into cache with isMRRT=true. How it used to work is, one doesn't have the iteration number when create the response, and the other one has 21(this is simply there is already 20 entries, and then 21 will be the newly added one). This works totally fine before since we look up cache by userid, clientid, and authority, however with the cross tenant refresh token, we will update the refresh token of the cached entry if there is a match of user id, and client id. That's why I change it to 21, just to make it the same with the entry added after it. 

I agree, will add the doc.